### PR TITLE
Do not expect redundancy value in CreatePool parameters

### LINKED ISSUE #2969: Remove redundancy option from PoolCreate method
it does nothing.

### PR DESCRIPTION
**Pull request by mulkieran**
Closes #2969

### CODE REVIEW

**Comment by mulkieran** [comment]
All stratis-cli related tests should fail.

**Comment by mulkieran** [comment]
@jbaublitz I'm not convinced that abstracting the shared content of ```create_pool()``` will do any good over the long term, but I'm not really opposed to doing it, either.

File: src/dbus_api/api/manager_3_5/methods.rs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    let base_path = m.path.get_name();
+    let message: &Message = m.msg;
+    let mut iter = message.iter_init();
+
+    let name: &str = get_next_arg(&mut iter, 0)?;
+    let devs: Array<'_, &str, _> = get_next_arg(&mut iter, 2)?;
+    let (key_desc_tuple, clevis_tuple): EncryptionParams = (
+        Some(get_next_arg(&mut iter, 3)?),
+        Some(get_next_arg(&mut iter, 4)?),
+    );
```
Comment (jbaublitz) [inline src/dbus_api/api/manager_3_5/methods.rs:37]: It looks like the arg count is off by 1.

**Review by jbaublitz — CHANGES_REQUESTED** [review]
Just one issue I'm seeing. I don't think sharing the code is really necessary here either.

File: src/dbus_api/api/manager_3_5/methods.rs (hunk 1 above)
Comment (mulkieran) [reply src/dbus_api/api/manager_3_5/methods.rs]: Fixed, thanks.

**Review by jbaublitz — APPROVED** [review]

**Pull request merged**